### ISSUE TITLE
Disable external access, does not remove external_public_agent_hostname requirement

### ISSUE THREAD

**Issue by a-nldisr** [dcos-labs/dcos-jupyterlab-service#18]
Setting external_access enabled to false does not remove the requirement for a filled in external_public_agent_hostname.

JSON snippet example
```
  "external_access": {
      "enabled": false,
      "external_public_agent_hostname": "shouldnothavetofillthisin.com"
    }
```

Removing the external_public_agent_hostname gives:
```
Expecting a string here
```

**Comment by vishnu2kmohan** [comment]
This is mandatory for now and will be optional in the next release.

**Comment by vishnu2kmohan** [comment]
@a-nldisr We've released a new version [`1.3.0-0.35.4`](https://github.com/dcos-labs/dcos-jupyterlab-service/releases/tag/1.3.0-0.35.4) of the service and it's now renamed to `beta-mesosphere-jupyter-service` in the DC/OS Catalog 

Please give this new version a spin and let me know if it resolves your issues.
